tests/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -371,4 +374,57 @@ public function testShorthandInForeignKeyReferenceWithMultipleColumns(): void
             $createTableTrackSql,
         );
     }
+
+    public function testListTableNoSchemaEmulation(): void
+    {
+        $this->dropTableIfExists('`list_table_no_schema_emulation.test`');
+
+        $ddl = <<<'DDL'
```

Comment:
Let's not clutter the reader's mind with one-time, inconsistently named variables.

```suggestion
        $this->connection->executeStatement(<<<'DDL'
```

Reply:
done